packages/protocol/migrationsConfig.js

```diff
@@ -238,6 +238,10 @@ const DefaultConfig = {
     commission: 0.1,
     votesRatioOfLastVsFirstGroup: 2.0,
   },
+  stableTokenRegistry: {
+    fiatTicker: '0x494e52', // fiat ticker
+    stableTokenContractName: '0x537461626c65546f6b656e494e52', // stable token contract name
```

Comment:
Developers should be able to read this file

Reply:
Or maybe leave this empty, as the new registry has the stabletokens harcoded and no new one is added.